pkg/output/out.go

```diff
@@ -0,0 +1,129 @@
+package output
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/arr-ai/arrai/pkg/ctxfs"
+	"github.com/arr-ai/arrai/rel"
+	"github.com/spf13/afero"
+)
+
+// HandleEvalOut handles output writing for evaluated values.
+func HandleEvalOut(ctx context.Context, value rel.Value, w io.Writer, out string) error {
```

Comment:
This seems to be just outputting, not "handling" output. Perhaps call it `OutputValue`.